challenge-07/challenge-07.js

```diff
@@ -54,7 +63,12 @@ Só vamos mudar o range:
 - No segundo "for", mostre os números ímpares entre 111 e 125, inclusive eles.
 */
 console.log( 'Números pares entre 100 e 120:' );
-// ?
+for( var num = 100; num <= 120; num++){
+	num % 2 === 0 ? console.log(num++) : '';
```

Comment:
-1
Cuidado com isso aqui! No `for`, você já diz que, a cada iteração, o `num` será incrementado. Se você incrementa aqui também, você acaba duplicando a incrementação. 

Aí você me pergunta: _Então por que isso funcionou?_
Porque você está fazendo a incrementação uma vez sim, e uma vez não, já que o JS só passa por esse `console.log` se o número dividido por 2 tiver resto zero.

Compreende o problema?